boards/nxp/imx95_evk/board.cmake

```
@@ -3,3 +3,12 @@
 if (CONFIG_SOF AND CONFIG_BOARD_IMX95_EVK_MIMX9596_M7_DDR)
   board_set_rimage_target(imx95)
 endif()
+
```

Include the following items by CONFIG_BOARD_NXP_SPSDK_IMAGE
```suggestion
if (CONFIG_BOARD_NXP_SPSDK_IMAGE)
```

---

no space between if and ( in cmake

---

Get it. Thanks.